recipes/pynumero_build/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pynumero_build" %}
+{% set version = "0.1" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/santiagoropb/pynumero_build/archive/v{{ version }}.tar.gz
+  sha256: 8626219245936b7f4623aa0ae71deb154814bd8bd2bc0f10d55c5c7019b32644
+  
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
+  
+requirements:
+  build:
+    - cmake
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - ampl-mp
+  run:
+    - ampl-mp
+
+test:
+  files:
+    - test
+    - simple_nlp.nl
+    
+about:
+  home: https://github.com/santiagoropb/pynumero_build.git
+  license: BSD
```

Comment:
Please, add the ``license_file``

Reply:
Sure I will add the file. I have a question about the output files in te recipe. My project builds two shared libraries and an executable
libpynumero_ASL.so
libpynumero_SPARSE.so
asl_test
CI is building correctly the libraries and the executable but conda is not finding them and its giving the warning 
WARNING: No files or script found for output pynumero_build
Is there a way to specify in the recipe that I want those three files in the output?

Thanks!

Reply:
An easy way is just to copy your shared objects into ``$PREFIX/lib`` in your build script

Reply:
Just the second one may be a problem

Try to add ``ampl-mp`` in run requirements.

But I think that's fine. I will mark @ocefpaf to also take a look on that

Reply:
Let's call the experts @isuruf and @msarahan here :smile:

Reply:
Thanks Marcelo and Filipe,

Initially I was planning on distributing the libraries by precompiling them so I am compiling with -static-libstdc++. I wonder if I can get rid of that flag now and let conda export the libc++ together with my libraries.

Reply:
Both warnings can be ignored since we are using toolchain for now.

Reply:
I just added the license file to the source. Seen that the build is passing I believe the pull request can be merged?